01.md

```diff
@@ -1,177 +1,235 @@
 NIP-01
 ======
 
-Basic protocol flow description
--------------------------------
+Nostr Protocol
```

Comment:
```suggestion
Basic Protocol
```

The title "Nostr Protocol" is pretty cool, but it is meaningless, because I think all NIPs are descriptions of the Nostr protocol.

Reply:
Yeah. I was trying to find a way to say that this is the only mandatory part everyone must code. Everything else is optional. "The Basic Flow" before wasn't doing it. 

The other idea was to avoid the assumption that the "Nostr Protocol" includes all NIPs, which I think makes it too big for our own good.

Reply:
> Yeah. I was trying to find a way to say that this is the only mandatory part everyone must code. Everything else is optional. "The Basic Flow" before wasn't doing it. 
> 
> The other idea was to avoid the assumption that the "Nostr Protocol" includes all NIPs, which I think makes it too big for our own good. 
> 
> 

This.

NIP-01 is the protocol and the other NIPs are optional conventions. One could even say, _layers_, built on top of NIP-01

Reply:
Yeah, that make sense to me. Thanks.